HSDP-1D.md

```diff
@@ -0,0 +1,89 @@
+```
+                              __
+                 ____________ ____
+             _______________  _________
+          _________________  ______________
+       ___________________  __________________                 __________________
+     _____________________  ________________________        ________________________
+    _______________________  ______________    _____          _______      __________
+   _________________________  ________________     __    ________________    ________
+  ___________________________ ____         _______      _______    ________   _______
+  ____________________        _                ___        ________    _____    ______
+  _________________                                        _________   _____   ______
+  ________________                                         _________   _____   ______
+  ________________                                       ___________   ____   ______
+  ________________                                   ______________    ____  ______
+  _________________                              _________________    ___   _____
+   ____________________                    ______________________    __   ____
+    __________________________________________________________
+     _______________________________________________________
+       __________________________________________________
+         ____________________________________________
+            ____________________________________
+                 _______________________
+
+-----------------------------------------------------------------------------------------
+Hardware Info : 
+-----------------------------------------------------------------------------------------
+Device Name       : High Speed Data Printer Model 1Data (HSDP-1D)
+Device ID         : TBD
+Device Type       : TBD
+Manufacturer Name : SlapDragon
+Manufacturer ID   : 0xF6976D00
+
+-----------------------------------------------------------------------------------------
+Description :
+-----------------------------------------------------------------------------------------
+The HSDP line is all about printing as fast as possible the data you need.
```

Comment:
Phrasing awkward? maybe "The HSDP line is all about printing the data you need as fast as possible ."?